plugin/pkg/scheduler/core/extender.go

```diff
@@ -153,6 +155,19 @@ func (h *HTTPExtender) Prioritize(pod *v1.Pod, nodes []*v1.Node) (*schedulerapi.
 	return &result, h.weight, nil
 }
 
+// Bind informs the extender about the scheduling decision, before the decision is
+// conveyed to the apiserver.
+func (h *HTTPExtender) Bind(pod *v1.Pod, node string) error {
+	if h.bindVerb == "" {
+		return nil
+	}
+	binding := &schedulerapi.Binding{
+		Pod:  *pod,
+		Node: node,
+	}
+	return h.send(h.bindVerb, &binding, nil)
```

Comment:
Thanks @wanghaoran1988 I missed that. Added a check and an integration test case for bind error.